### PR TITLE
Add basic auth

### PR DESCRIPTION
**Pull request by adrianz-kainos**

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -1,6 +1,31 @@
-var express = require('express'),
-    router  = express.Router(),
-    _       = require('underscore');
+var express 	= require('express'),
+    router  	= express.Router(),
+    fs          = require('fs'),
+	basicAuth 	= require('basic-auth'),
```
Comment (xjedam) [inline app/routes.js]: would be best to use the same convention of using spaces to prevent indentation mess

File: app/routes.js [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+router.use(function (req, res, next) {	
+	  function unauthorized(res) {
+	    res.set('WWW-Authenticate', 'Basic realm=Authorization Required');
+	    return res.send(401);
+	  };
+	  var user = basicAuth(req);
+	  if (!user || !user.name || !user.pass) {
+	    return unauthorized(res);
+	  };
+	  if (user.name === storedUser.login && user.pass === storedUser.password) {
```
Comment (xjedam) [inline app/routes.js]: Couldn't we collapse those two if statements into one? Sth like if (user && user.name === ...

Comment (adrianz-kainos) [inline app/routes.js]: Done.

File: app/routes.js (hunk 1 above)
Comment (adrianz-kainos) [inline app/routes.js]: Done.

**Pull request merged**